gamestonk_terminal/etf/yfinance_view.py

```diff
@@ -0,0 +1,100 @@
+"""Yahoo Finance view"""
+__docformat__ = "numpy"
+
+import os
+import pandas as pd
+from matplotlib import pyplot as plt
+from tabulate import tabulate
+from gamestonk_terminal.etf import yfinance_model
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.config_plot import PLOT_DPI
+from gamestonk_terminal.helper_funcs import plot_autoscale, export_data
+
+
+def display_etf_weightings(
+    name: str,
+    raw: bool = False,
+    min_pct_to_display: float = 5,
+    export: str = "",
+):
+    """Display sector weightings allocation of ETF. [Source: Yahoo Finance]
+
+    Parameters
+    ----------
+    name: str
+        ETF name
+    raw: bool
+        Display sector weighting allocation
+    min_pct_to_display: float
+        Minimum percentage to display sector
+    export: str
+        Type of format to export data
+    """
+    sectors = yfinance_model.get_etf_sector_weightings(name)
+    if not sectors:
+        print("No data was found for that ETF\n")
+        return
+
+    holdings = pd.DataFrame(sectors, index=[0]).T
+
+    title = f"Sector holdings of {name}"
+
+    if raw:
+        print(f"\n{title}")
+        holdings.columns = ["% of holdings in the sector"]
+        if gtff.USE_TABULATE_DF:
+            print(
+                tabulate(
+                    holdings,
+                    headers=holdings.columns,
+                    showindex=True,
+                    tablefmt="fancy_grid",
+                ),
+            )
+        else:
+            print(holdings.to_string, "\n")
+
+    else:
+        main_holdings = holdings[holdings.values > min_pct_to_display].to_dict()[
+            holdings.columns[0]
+        ]
+        if len(main_holdings) < len(holdings):
+            main_holdings["Others"] = 100 - sum(main_holdings.values())
+
+        legend, values = zip(*main_holdings.items())
+
+        plt.subplots(figsize=plot_autoscale(), dpi=PLOT_DPI)
```

Comment:
havent we switched to using fig, ax everywhere?

Reply:
I'm not returning fix, ax because I'm not using them anywhere after. Not sure what you mean here, may be missing something.

Reply:
We have just been using` fig, ax = plt.subplots()` throughout the code instead of just calling through plt.  Doesn't really mater, but can make life easier if one wants to return an axes object in a bigger plot down the road.

Reply:
I haven't actually be using that because I didn't know. Can we have a call next week to discuss this and Matplotlib overall style? To make sure we have consistency across, and make some important decisions are made and we tackle them all at once. @piiq

Reply:
lol yeah we should have a consistent format across.